github-actions/summarize-errors/package.json

```diff
@@ -14,7 +14,8 @@
 	"dependencies": {
 		"@actions/core": "^1.2.4",
 		"@actions/github": "^5.0.0",
-		"himalaya": "^1.1.0"
+		"himalaya": "^1.1.0",
+		"xml-js": "^1.6.11"
```

Comment:
No 3PP necessary, since we already use this in the analyzer itself. (See `RuleResultRecombinator.ts`)